lib/mrsk/cli/healthcheck.rb

```diff
@@ -11,36 +15,33 @@ def perform
       begin
         execute *MRSK.healthcheck.run
 
-        target = "Health check against #{MRSK.config.healthcheck["path"]}"
         attempt = 1
-
         begin
-          status = capture_with_info(*MRSK.healthcheck.curl)
-
-          if status == "200"
-            info "#{target} succeeded with 200 OK!"
+          case output = capture_with_info(*MRSK.healthcheck.curl, raise_on_non_zero_exit: false)
```

Comment:
We were raising on non-200-OK curl responses previously, bypassing the status parsing conditions below.

Instead, don't raise, eliminating `SSHKit::Command::Failed` handling, giving us just retries on `HealthcheckError` to worry about.